app/mixins/message-wall-post-notification.js

```diff
@@ -0,0 +1,44 @@
+import Mixin from '@ember/object/mixin';
+
+export default Mixin.create({
+  /**
+  * Constructs a localized message wall post notification body
+  * @param {Ember.Object} model
+  * @returns {string}
+  */
+  getMessageWallPostBody(model) {
+    const firstReplierName = model.get('latestActors.0.name')
+    const postTitle = this.postTitleMarkup;
+
+    // need this data !!
+    const isOwnWall = firstReplierName === '// check model.uri for username? in Message Wall uri';
+    const postRemoved = '// need check to see if post has been removed';
+
+    // check for is own wall
+    if (isOwnWall) {
+        // check for is post removed
+        if (postRemoved) {
```

Comment:
Move this stuff to another file for the `message-wall-post-removed` type.

Reply:
Separated into it's own file `message-wall-post-removed-notification.js`.